### ISSUE TITLE
Docs build failure (API docs gen)

### ISSUE THREAD

**Issue by zmoon** [NOAA-CSL/MELODIES-MONET#97]
Noticed in #96 

```
Warning, treated as error:
/home/runner/work/MELODIES-MONET/MELODIES-MONET/docs/api/melodies_monet.rst:26:autosummary: stub file not found 'melodies_monet.melodies_monet.driver'. Check your autosummary_generate setting.
```

This can happen when a submodule imports the top-level module (https://github.com/sphinx-doc/sphinx/issues/8963), but we don't have that afaict. And it was working in [the last develop CI run](https://github.com/NOAA-CSL/MELODIES-MONET/actions/runs/2833757405), with the same Sphinx version as in this failure (4.5.0). So I am pretty confused.

I was able to reproduce locally only after recreating the conda environment.

The conda environment before this failure is not identical to when it starting failing, but I haven't determined yet what could have caused this.

<details><summary>Differences in versions</summary>

| name             | version_old   | version_new   |
|:-----------------|:--------------|:--------------|
| asttokens        | 2.0.5         | 2.0.8         |
| executing        | 0.9.1         | 0.10.0        |
| giflib           | 5.2.1         | -             |
| libsqlite        | -             | 3.39.2        |
| libwebp          | 1.2.3         | -             |
| libwebp-base     | 1.2.3         | 1.2.4         |
| nbconvert        | 6.5.2         | 6.5.3         |
| nbconvert-core   | 6.5.2         | 6.5.3         |
| nbconvert-pandoc | 6.5.2         | 6.5.3         |
| openjpeg         | 2.4.0         | 2.5.0         |
| partd            | 1.2.0         | 1.3.0         |
| pytz             | 2022.1        | 2022.2.1      |
| pyzmq            | 23.2.0        | 23.2.1        |
| setuptools       | 63.4.2        | 65.0.0        |
| stack_data       | 0.3.0         | 0.4.0         |
| xz               | 5.2.5         | 5.2.6         |
</details>

**Comment by zmoon** [comment]
This is fixed in develop (#98).

**Comment by rschwant** [comment]
These changes have been merged to main on April 14, 2023, so I'm closing this issue.
